app/services/import_prisoners.rb

```diff
@@ -46,6 +46,43 @@ def remove_previous_imports
   end
 
   def process_import_file
```

Comment:
I think the next step is to move this logic (CSV parsing, Prisoner record creation) into another service object. One that can be called via a rake task or by the ImportPrisoners service. All good for now.